### PR TITLE
TJ first updates to README.md

### PR DESCRIPTION
**Pull request by trjaffe**
Answering a few questions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,23 +12,26 @@ The Fornax Science Console is a compute system in the cloud near to NASA data in
...
 * Limits: What does it not do?
   * How many cores/RAM do I get?
+    * There are several options for the size of the compute.  Please select the smallest that you can use for testing and exploration.  Do not use the larger images unless you have already tested a smaller subset of the analysis on a smaller compute instance.    
     * In JK's understanding (which needs to be updated) you get what the numbers say when you choose a server upon login, but for a limited time if you need slightly more, the code will not crash but will have access to slightly more. 
     * Raen has seen some evidence of this "bursting" behavior, but doesn't have direct knowledge of the actual configuration. It would be good to know.
   * How much disk space do I have access to?
     * Current default is 10GB (Feb 2024).
+    * This can be increased on request.  
   * What restrictions are there to prevent egress charges?
     * Raen's understanding is:
```
Comment (jkrick) [inline README.md:25]: Now that we have a succinct answer from Tess, I think we can remove the historical aside and some other info that might be too much for users. 
```suggestion
   
```

File: README.md [hunk 2]
```diff
@@ -117,11 +126,12 @@ The Fornax Science Console is a compute system in the cloud near to NASA data in
   * follow [conda documentation](https://docs.conda.io/projects/conda/en/latest/user-guide/getting-started.html)
   * specifically [managing environments](https://docs.conda.io/projects/conda/en/latest/user-guide/getting-started.html#managing-envs)
 * Can I get a list of what software is already pre-installed on the Fornax Science Console?
+ * Software is installed in miniconda environments.  You can use the usual conda commands to list the contents of each.
```
Comment (jkrick) [inline README.md:129]:
```suggestion
 * Software is installed in miniconda environments.  You can use "[conda list](https://conda.io/projects/conda/en/latest/commands/list.html)"  to list the contents of each.  
```

**Review by jkrick — APPROVED** [review]
Thanks @trjaffe!  Some suggestions below, otherwise looks great.

**Review by zoghbi-a — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -12,23 +12,26 @@ The Fornax Science Console is a compute system in the cloud near to NASA data in
...
         * As far as I can tell, the best (AWS) option for actual controls -- beyond just "monitoring" support -- is [AWS Budgets Actions](https://aws.amazon.com/blogs/aws-cloud-financial-management/get-started-with-aws-budgets-actions/).
         * AWS may charge *ingress* fees to bring data into an SMCE pod or user instance. This would be completely separate from any egress fees. There Someone working more directly on Fornax Daskhub would need to answer whether/how ingress applies.
```
[the remark this answers is not in the document]
Comment (trjaffe) [reply README.md:29]: Lots of this info is still useful.  I would leave it.  (Except for the ingress, which we need an answer to.  My understanding is that there are no ingress charges.)

**Comment by trjaffe** [comment]
Merging this to get it on the site ASAP.  Will obviously need continuing updating.

**Pull request merged**